templates/index.ts.hbs

```diff
@@ -1,5 +1,10 @@
 {{#each children}}
-export { {{name.upperCamelCase}} } from "./{{name.lowerCamelCase}}";
+{{!-- We intentionally added the @ts-ignore comment --}}
+{{!-- This comment is used to ignore the error ts-2308 specifically. --}}
+{{!-- Reason: The API specs contain duplicated type declarations. --}}
+{{!-- rollup=plugin-ts does de-duplication, so it's okay to have dups --}}
+// @ts-ignore
+export * from "./{{name.lowerCamelCase}}";
```

Comment:
Good suggestion. I've updated the PR to use the namespaced types.

Reply:
@wjhsf does namespacing types this way have any advantages over explicitly exporting namespaces from within the clients?

Reply:
Using the native module system to create namespaces is preferred over the namespace keyword. (See [TypeScript Features to Avoid](https://www.executeprogram.com/blog/typescript-features-to-avoid).)